Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -283,7 +283,7 @@ codeunit 20352 "Connectivity App Definitions"
         AppDescription := 'Bank statement import & reconciliation.';
         AppProviderSupportURL := 'https://bankfeed.com/banks/';
         AppSourceUrl := 'https://appsource.microsoft.com/en-us/product/dynamics-365-business-central/PUBID.softera_baltic%7CAID.softeradokubank%7CPAPPID.74689c09-2ed3-4e69-a2e8-c9310a271b9a';
-        AppApprovedFor := 'LT,DK,HU,GB,ES,LV,DE,RO,IE,US';
+        AppApprovedFor := 'LT,DK,HU,GB,ES,LV,DE,RO,IE,US,EE,CZ,PL';
         AppWorksOn := 'W1,DK,GB,ES,DE,US';
```

Comment:
Should CZ not be added here too? EE and PL are covered under W1, but CZ has its own localization.
```suggestion
        AppWorksOn := 'W1,DK,GB,ES,DE,US,CZ';
```

Reply:
@JesperSchulz You are right, Jesper. Good catch.